src/main/java/com/github/manolo8/darkbot/backpage/hangar/Hangar.java

```diff
@@ -0,0 +1,59 @@
+package com.github.manolo8.darkbot.backpage.hangar;
+
+import com.google.gson.JsonDeserializationContext;
+import com.google.gson.JsonDeserializer;
+import com.google.gson.JsonElement;
+import com.google.gson.JsonParseException;
+import com.google.gson.annotations.SerializedName;
+
+import java.lang.reflect.Type;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+public class Hangar {
+    @SerializedName("hangarID") private int hangarId;
+    @SerializedName("hangar_is_active") private boolean hangarActive;
```

Comment:
```suggestion
    @SerializedName("hangar_is_active") private boolean isActive;
```
or maybe even just `active`